style.css

```diff
@@ -67,28 +72,239 @@ section {
+  padding-top: 0.1em;
+}
+
+.tags button {
+  display: flex;
+  list-style: none;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  border: none;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.project-detail {
+  width: 100px;
+  height: 40px;
+  border-width: 1px;
+  border-color: #396df2;
+  border-radius: 0.5em;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.04em;
+  color: #396df2;
+  background: #fff;
+}
+
+.about-myself {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: no-repeat;
+  padding: 16px;
+  margin: 20px;
+  width: 327px;
+  height: 1000px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.about-myself-title {
+  width: 327px;
+  height: 101px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 1px;
+}
+
+.about-myself-content {
+  width: 280px;
+  height: 120px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 1px;
+}
+
 .links {
   font-style: normal;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
-  padding: 0 24px;
+  margin-bottom: 1px;
 }
 
 .socials {
   display: flex;
   list-style: none;
-  padding: 0 12px;
+  padding: 0 1px;
 }
 
 .socials li {
-  padding-left: 8px;
+  padding-left: 16px;
 }
 
 .socials a {
   text-decoration: none;
   font-size: 22px;
   color: #505f79;
-  padding-left: 2px;
+}
+
+.resume {
+  width: 130px;
+  height: 44px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border-width: 1px;
+  border-color: #396df2;
+  color: #396df2;
+  border-radius: 0.5em;
+  background: #fff;
+}
+
+.languages {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  padding-top: 5px;
+}
+
+.skills {
+  display: flex;
+  list-style: none;
+  flex-direction: column;
+  padding-left: 4px;
+}
+
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  width: 280px;
+  background-color: #dfe1e6;
+  height: 1px;
+  margin: 6px;
+}
+
+.frameworks {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
 }

```

Comment:
- [ ] **OPTIONAL** 
This table summarizes where your design has to be improved. Please make sure You stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

| Your Design |Figma Design|
| :---:   | :---: | 
|Check icon at the right|
| ![noicon](https://user-images.githubusercontent.com/26736582/198749501-cc502298-9bd7-495e-8cf1-64fb884f4dc6.png) | ![iconpresent](https://user-images.githubusercontent.com/26736582/198749514-eca6c8f6-9b21-4bab-aba8-c623cda449a2.png) |
| ![langauge list](https://user-images.githubusercontent.com/26736582/198749606-bd1ac8e7-7450-4377-9ea3-6565d30460a7.png) |![lang-figmalist](https://user-images.githubusercontent.com/26736582/198749628-b830b42c-8565-47ae-bec4-9e423b1c10de.png) |
| ![arrowmiss](https://user-images.githubusercontent.com/26736582/198749828-9bd7efe0-d3ac-4b8c-abe9-b4fb1ae37887.png) |  ![arrowthere](https://user-images.githubusercontent.com/26736582/198749937-009d881a-2c66-46b4-bfdc-d52d3cce1ed4.png)|
| ![nospace](https://user-images.githubusercontent.com/26736582/198749960-17bbc118-5dee-4926-9af9-c3ecb1bff535.png) | ![spacepresent](https://user-images.githubusercontent.com/26736582/198749986-3b9d70c8-c0e2-4dae-8e3c-c954a31ef363.png) |